tapdb/assets_store.go

```diff
@@ -1572,6 +1575,8 @@ func (a *AssetStore) importAssetFromProof(ctx context.Context,
 	// As a final step, we'll insert the proof file we used to generate all
 	// the above information.
 	scriptKeyBytes := newAsset.ScriptKey.PubKey.SerializeCompressed()
+	a.Lock()
```

Comment:
nit: move this to above the comment into its own visual block (empty line before and after) to stand out more clearly?

Reply:
I guess this will be removed by the next commit anyway, but generally we try to make the use of mutexes stand out visually.